NotebookAnalyzers/utils.py

```diff
@@ -0,0 +1,15 @@
+import json
+def notebookToCode(filename):
+    """The function takes the name of the desired .ipynb file in the target notebooks folder and return a python code string"""
+    f=open("../TargetNotebooks/"+(filename),)
+    data = json.load(f)
+    code=[]
+    for cell in data["cells"]:
+        if cell["cell_type"] == 'code':
+            code.append(''.join(cell["source"]))
+    code = ''.join(code)
```

Comment:
Userei un carattere di line feed (`\n`) per separare una cella di codice dall'altra (piuttosto che la stringa vuota). In questo modo ci assicuriamo di non introdurre artificialmente errori sintattici in caso di celle che non finiscono con un'andata a capo.